packages/next/test/fixtures/00-app-dir/vercel.json

```diff
@@ -12,15 +12,26 @@
       "mustContain": "hello from app/dashboard"
     },
     {
-      "path": "/dashboard/another",
+      "path": "/dashboard",
```

Comment:
Lets changes this file from `vercel.json` to `probes.json` and remove the `builds` prop

Reply:
We need the builds for the integration test still